woocommerce/woocommerce-gutenberg-products-block
# Use wc_review_ratings_enabled() API function to check if review ratings are enabled

Found this while working on #1298: when we are checking if review ratings are enabled, we are reading the `woocommerce_enable_review_rating` option, but WooCommerce has a specific function that is more reliable because it also checks `woocommerce_enable_reviews` (see the [code here](https://docs.woocommerce.com/wc-apidocs/source-function-wc_reviews_enabled.html#438)).*

This PR replaces our check, from reading the option directly, to use WooCommerce's function. In parallel to that, I renamed the setting from `ENABLE_REVIEW_RATING` to `REVIEW_RATINGS_ENABLED`, I think it's better to stick closer to WooCommerce API than to names used by options in the database.

\* That function was introduced in WC 3.6 and we require >=3.7.

### How to test the changes in this Pull Request:

1. Create a post with an _All Reviews_ block and make sure you have at least one review with rating.
2. Go to _WooCommerce_ > _Settings_ > _Products_  and disable _Enable star rating on reviews_.
3. Reload the post and verify rating stars are no longer displayed.

### Changelog

> Devs: `ENABLE_REVIEW_RATING` setting was renamed to `REVIEW_RATINGS_ENABLED` and now it also verifies reviews are enabled, to better match WooCommerce API.
